bin/update_pythons.py

```diff
@@ -281,7 +283,7 @@ def update_pythons(force: bool, level: str) -> None:
 
     if original_toml == result_toml:
         rich.print("[green]Check complete, Python configurations unchanged.")
-        exit()
+        return
```

Comment:
exit() should never be used in a file. It is only defined if site is imported (same with `quit()`). Either use sys.exit, or in this case, just return, we are in a main function already. :)

Reply:
good to know!

Reply:
I have realized it's not the only offender. :)

Reply:
#539